files-community/Files
# Fix: Fixed network locations not appearing on the sidebar

**Resolved / Related Issues**
   Closes #issue...

Since becoming a view model, UpdateDrivesAsync() of NetworkDrivesViewModel was not being called anymore.

**Validation**
How did you test these changes?
   1. Connect a network resource (e.g ftp server)
   2. Verify that it appears in the sidebar

**Screenshots (optional)**
Add screenshots here.
![image](https://github.com/files-community/Files/assets/9673091/7bc1f968-42aa-426d-a1ef-d7fb89cc134f)
